core/src/test/java/org/elasticsearch/index/query/TemplateQueryIT.java

```diff
@@ -144,28 +136,32 @@ public void testTemplateInFile() {
         assertHitCount(sr, 2);
     }
 
-    @Test
-    public void testRawEscapedTemplate() throws IOException {
-        String query = "{\"template\": {\"query\": \"{\\\"match_{{template}}\\\": {}}\\\"\",\"params\" : {\"template\" : \"all\"}}}";
-
-        SearchResponse sr = client().prepareSearch().setQuery(query).get();
-        assertHitCount(sr, 2);
-    }
-
-    @Test
-    public void testRawTemplate() throws IOException {
-        String query = "{\"template\": {\"query\": {\"match_{{template}}\": {}},\"params\" : {\"template\" : \"all\"}}}";
-        SearchResponse sr = client().prepareSearch().setQuery(query).get();
-        assertHitCount(sr, 2);
-    }
-
-    @Test
-    public void testRawFSTemplate() throws IOException {
-        String query = "{\"template\": {\"file\": \"storedTemplate\",\"params\" : {\"template\" : \"all\"}}}";
-
-        SearchResponse sr = client().prepareSearch().setQuery(query).get();
-        assertHitCount(sr, 2);
-    }
+    // NORELEASE These need to be tested in TemplateQueryBuilderTests
```

Comment:
not sure about these....they are integration tests, testing that a template search returns results. What did you have in mind?